src/testset_bpdn.jl

```diff
@@ -3,14 +3,18 @@ export setup_bpdn_l0, setup_bpdn_l1, setup_bpdn_B0
 
 function setup_bpdn_l0(args...; kwargs...)
   model, nls_model, _ = bpdn_model(args...; kwargs...)
-  λ = norm(grad(model, zeros(model.meta.nvar)), Inf) / 10
+  y = similar(model.meta.x0)
+  grad!(model, y, zeros(model.meta.nvar))
+  λ = norm(y) / 10
   h = ProximalOperators.NormL0(λ)
   return RegularizedNLPModel(model, h), RegularizedNLSModel(nls_model, h)
 end
 
 function setup_bpdn_l1(args...; kwargs...)
   model, nls_model, _ = bpdn_model(args...; kwargs...)
-  λ = norm(grad(model, zeros(model.meta.nvar)), Inf) / 10
+  y = similar(model.meta.x0)
+  grad!(model, y, zeros(model.meta.nvar))
+  λ = norm(y) / 10
```

Comment:
@dpo  The main change I made here was to avoid the error where ```grad``` is not imported in the package, while ```grad!``` is.